examples/web-server-da/README.md

```diff
@@ -0,0 +1,29 @@
+Commands to run da examples: 
```

Comment:
I forgot to build before running the examples, and got a `no example target` error--might be something that's trivial but worth mentioning here.